auth-web/src/components/auth/BusinessContactForm.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div>
+    <v-form ref="form" lazy-validation>
+      <v-expand-transition>
+        <div class="business-contact-form_alert-container" v-show="formError">
+          <v-alert
+            :value="true"
+            color="error"
+            icon="warning"
+          >{{formError}}
+          </v-alert>
+        </div>
+      </v-expand-transition>
+    <div class="business-contact-form_row">
+      <v-text-field
+        box
+        label="Email Address"
+        req
+        persistent-hint
+        :rules="emailRules"
+        v-model="emailAddress"
+      >
+      </v-text-field>
+    </div>
+    <div class="business-contact-form_row">
+      <v-text-field
+        box
+        label="Confirm Email Address"
+        req
+        persistent-hint
+        :error-messages="emailMustMatch()"
+        v-model="confirmedEmailAddress"
+      >
+      </v-text-field>
+    </div>
+    <div class="business-contact-form_row">
+      <v-layout wrap>
+        <v-flex xs6 class="mr-5">
+          <v-text-field
+            box
+            label="Phone e.g. (555)-555-5555"
+            persistent-hint
+            type="tel"
+            v-mask="['(###)-###-####']"
```

Comment:
do you wanna consider moving this to a common place? a constant class for masks..low priority

Reply:
Phone number is not necessary to be this format if they are out of Canada.

Reply:
@saravankumarpa Yeah will look at moving to common components.

@mengdong19 That's true... I did try to include masking options for different regions but it was very difficult.  Will talk to the UX guys about strategies for this.